modules/apigee_edge_teams/src/Entity/ListBuilder/TeamListBuilder.php

```diff
@@ -110,4 +110,18 @@ public function buildRow(EntityInterface $entity) {
     return $row + parent::buildRow($entity);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function render() {
+    $build = parent::render();
+
+    // Team lists vary for each user.
+    $build['#cache']['contexts'][] = 'user';
+    $build['#cache']['tags'][] = 'user:' . \Drupal::currentUser()->id();
+    $build['#cache']['max-age'] = 0;
```

Comment:
This kills the cache, so the above settings does nothing...

Reply:
This brings up an interesting point. If we only cache per user context, then the cache will not be invalidated if a user's role is edited to have different permissions (for example, given the "administer teams" permission), or if team roles are edited. I just pushed an update that takes that into account, as well as the max-age from the team caching configuration.